repositories/dbMovieRepository.js

```diff
@@ -0,0 +1,163 @@
+const { Movie } = require('../models');
+const { Op, literal } = require('sequelize');
+const { sequelize } = require('../models');
+
+class DBMovieRepository {
+  async paginateData(data, page, limit) {
```

Comment:
Confirm if needed. If not, lets delete it

Reply:
Modified, since it helps eliminating repition